programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(n) {
+    
+    const strNum = String(n)    
```

Comment:
```suggestion
    const convertStr = String(n)    
```
변수명은 길어져도 확실히 무슨 일을 하는지 알려주는 것이 좋은거 같아요